packages/mui-joy/src/Alert/AlertProps.ts

```diff
@@ -8,13 +9,44 @@ export interface AlertPropsVariantOverrides {}
 export interface AlertPropsColorOverrides {}
 export interface AlertPropsSizeOverrides {}
 
+interface ComponentsProps {
+  root?: SlotComponentProps<
+    'div',
+    { component?: React.ElementType; sx?: SxProps },
+    AlertOwnerState
+  >;
+  startDecorator?: SlotComponentProps<
+    'span',
+    { component?: React.ElementType; sx?: SxProps },
+    AlertOwnerState
+  >;
+  endDecorator?: SlotComponentProps<
+    'span',
+    { component?: React.ElementType; sx?: SxProps },
+    AlertOwnerState
+  >;
+}
+
 export interface AlertTypeMap<P = {}, D extends React.ElementType = 'div'> {
   props: P & {
     /**
      * The color of the component. It supports those theme colors that make sense for this component.
      * @default 'primary'
      */
     color?: OverridableStringUnion<ColorPaletteProp, AlertPropsColorOverrides>;
+    /**
+     * Replace the default slots.
```

Comment:
Could you describe what they are instead of using an imperative sentence?

Reply:
Like the following?
```jsx
  /**
   * The components used for each slot inside the Alert.
   * Either a string to use a HTML element or a component.
   * @default {}
   */
   ```

Reply:
Done in this commit: [Improve API docs for slots prop](https://github.com/mui/material-ui/pull/34997/commits/10fc03df7a83d9ff498aebc30b8127859938353a)